jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -361,23 +366,7 @@ public RequestContext getRequestContext(Request baseRequest) {
           VmRuntimeUtils.ONE_DAY_IN_MILLIS, defaultEnvironment);
       final HttpOutput httpOutput = request.getResponse().getHttpOutput();
       final HttpOutput.Interceptor nextOutput = httpOutput.getInterceptor();
-      httpOutput.setInterceptor(new Interceptor() {
-        @Override
-        public void write(ByteBuffer content, boolean complete, Callback callback) {
-          requestSpecificEnvironment.clearTicket(); // Request ticket not valid after commit
```

Comment:
Help me understand. So, in the current code (where we don't buffer/delay the applications writes) this is going to clear the ticket after the first write to the servlet's outputstream?